spec/tree_spec.rb

```diff
@@ -1,14 +1,156 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
-  it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+describe AppleTree do
+
+  context "when first created" do
+    subject(:newAppleTree) { AppleTree.new }
```

Comment:
this usage is completely ok, but sometimes you'll see use of `described_class` instead. rubyists would probably prefer snake case over camel case too.